test/utils.test.js

```diff
@@ -122,3 +123,34 @@ QUnit.test('array-like', function(assert) {
   assert.ok(result.map);
   assert.deepEqual(result.length, 2);
 });
+
+QUnit.module('xml', {
+  beforeEach() {
+    this.fixture = document.createElement('div');
+
+    const foo = document.createElement('test');
+    const bar = document.createElement('div');
+    const baz = document.createElement('div');
+
+    foo.innerText = 'foo ';
+    bar.innerText = 'bar';
+    baz.innerText = 'baz';
+
+    this.fixture.appendChild(foo);
+    this.fixture.appendChild(bar);
+    this.fixture.appendChild(baz);
+  }
+});
+
+QUnit.test('findChildren', function(assert) {
+  // HTML tagName properties are always capitalized, xml are not
```

Comment:
Should we test based on XML instead of HTML then, since we use the functions for XML parsing?